### PR TITLE
WIP | Deprecate ch.lambdaj library and use JDK lamdas

### PR DESCRIPTION
**Pull request by rehammuzzamil**
Issue # 231
Remove dependency of ch.lambdaj library and use JDK lambas

### CODE REVIEW

File: src/main/java/org/opensrp/web/controller/ActionController.java [hunk 1]
```diff
@@ -53,24 +51,24 @@ public ActionController(ActionService actionService, ClientsRepository c, Alerts
     @ResponseBody
     public List<Action> getNewActionForANM(@RequestParam("anmIdentifier") String anmIdentifier, @RequestParam("timeStamp") Long timeStamp){
         List<org.opensrp.scheduler.Action> actions = actionService.getNewAlertsForANM(anmIdentifier, timeStamp);
-        return with(actions).convert(new Converter<org.opensrp.scheduler.Action, Action>() {
-            @Override
-            public Action convert(org.opensrp.scheduler.Action action) {
-                return ActionConvertor.from(action);
-            }
-        });
+        List<Action> actionList = new ArrayList<>();
```
Comment (maimoonak) [inline src/main/java/org/opensrp/web/controller/ActionController.java]: can we avoid this list ... may be something like https://stackoverflow.com/a/40035280

File: src/main/java/org/opensrp/web/security/DrishtiAuthenticationProvider.java [hunk 2]
```diff
@@ -100,15 +99,13 @@ public boolean supports(Class<?> authentication) {
...
-
-			@Override
-			public SimpleGrantedAuthority convert(String role) {
-				if (GET_ALL_EVENTS_ROlE.equals(role))
-					return new SimpleGrantedAuthority("ROLE_ALL_EVENTS");
-				return new SimpleGrantedAuthority("ROLE_OPENMRS");
-			}
-		});
+		List<SimpleGrantedAuthority> simpleGrantedAuthorities = new ArrayList<>();
+		if(user != null && user.getRoles()!=null) {
```
Comment (maimoonak) [inline src/main/java/org/opensrp/web/security/DrishtiAuthenticationProvider.java]: does not look like both are performing same function... if (GET_ALL_EVENTS_ROlE.equals(role))...

File: src/main/java/org/opensrp/web/controller/ActionController.java (hunk 1 above)
Comment (rehammuzzamil) [reply src/main/java/org/opensrp/web/controller/ActionController.java]: > can we avoid this list ... may be something like https://stackoverflow.com/a/40035280

Code refactoring done, removed creating new list code

File: src/main/java/org/opensrp/web/security/DrishtiAuthenticationProvider.java (hunk 2 above)
Comment (maimoonak) [reply src/main/java/org/opensrp/web/security/DrishtiAuthenticationProvider.java]: plz ignore this

**Pull request merged**